### DISCUSSION TITLE
为什么我部署的是英文呢？怎样能让它部署就是中文呢或者修改成中文

### DISCUSSION THREAD

**Discussion by aofly** [walinejs/waline#1905] [Q&A] [answered]
为什么我部署的是英文呢？怎样能让它部署就是中文呢或者修改成中文

**Comment by 37748846** [comment]


**Reply by lizheming** [reply]
这个问题之前 miss 掉了，现在回了。另外有问题就提，提了就等回答就好。一顿瞎诶特你不觉得很打扰其他人么，你真的觉得你这么做很尊重其他人吗？人与人的沟通一定要换位思考，你觉得重要且紧急的问题其他人并不认为。你可以提一些小白的问题，但你这随意诶特人打扰 Waline 用户的行为请下次不要再做了。

---

This question was missed before, but now it is returned. In addition, if you have any questions, just ask, and just wait for the answer. Don't you think it's very disturbing to others? Do you really think that you respect others by doing this? Communication between people must be empathetic, and other people do not think that you think important and urgent issues. You can ask some idiot questions, but please don't do it again next time, you random person disturbing Waline users.

**Reply by Mister-Hope** [reply]
> 怎么没人回答？我也纳闷 我部署的服务端为英文，所有提示都是英文 同求解决方法 @ aofly @ hex-ci @ lizheming @ stevygee @ eallion

暂不提我们的文档应该很显著的标明了多语言文档的位置并且提供了搜索以便你随便搜一下中文或者多语言就能找到，这个项目只有我和 @lizheming 两个维护者，你提醒我们我们可以接受，你这一串 @ 其他人你觉得是 OK 的么？

另外我不觉得我们的态度有问题，用爱发电不是道德捆绑的理由。想发就发不想发就不发。不提其他用户是否需要回答这个问题，我觉得文档有的东西我就没必要回答。lizheming 我不清楚，至少对于我来说，当初这个问题我是看到了的，我在文档随便搜了一下多语言我就看到了所以我就不是很想回。任何开源作者都没有义务在讨论区成为 Docs Provider，向你提供文档链接。提供详细文档是开源项目作者的义务，而仔细阅读项目文档应该是开源项目使用者的义务。

**Reply by lizheming** [reply]
> 我对您能回答问题，表示感谢，请问等待回答是多长时间？小白就没有权利问问题么？没及时回答可以，我也不会去追问，开源的项目都是自愿维护的，我感谢你们的付出，只是我觉得快一个月了没人回复问题，就不能提醒你们一下？不能@？好吧。你忙理解，我对打扰到你感到抱歉，希望你生活中不是这种这种态度。
> 最后谢谢回答

可以不用把回复删掉，留着让大家来评判吧。对这个内容我只想说，生活中我的态度比这个差多了。

**Reply by 37748846** [reply]
真的在上纲上线，我一直在说谢谢，还要让我怎么样？我还道歉了，不依不饶？最后我觉得我说的话可能会让你们认为挑衅，我还删除了。拿出来鞭尸？
不管怎样还是谢谢你们的付出。希望话题到此为止。

**Reply by lizheming** [reply]
实话实说，你的道歉我并没有看到就被你自己给删除掉了。你自己认为挑衅把内容删除掉了，就当做意念道歉了么。

我是在 GitHub 消息通知邮件中才捞到了上下文。对于这段回复，我觉得有必要放出来让大家都了解到全文客观评价下，所以我贴回来了。

另外，我想你根本没理解到问题出在哪里，只是觉得我的回复对你来说非常的冒犯。所以我再特地着重下这句话：

> 你可以提一些小白的问题，但你这随意诶特人打扰 **Waline 用户** 的行为请下次不要再做了。

然后我想说即使你不单独诶特我，只要你在 GitHub Discussion 中回复了，我是能收到通知的，因为身为作者仓库的所有活动我都是默认通知的。然后是你诶特的几位用户里，除了一个是提问的同学，一个是我之外，其他三个均不是 Waline 的开发者，都是 Waline 的使用者，一位用户我可以确认他已经不使用 Waline 了，还有一位甚至还是国外的用户，他甚至可能根本不知道发生了什么就被诶特了。为此我还特地写了一份英文的回复谨防对方一脸懵逼。所以你真的认为这种行为是没问题的吗。即使你的道歉也都是「对打扰到你感到很抱歉」，但这真的只是打扰到我么。希望补充了这些内容之后你能意识到自己的行为有多么的不礼貌吧。

**Comment by lizheming** [comment, marked as answer]
为了更好的服务多语言用户，Waline 会默认使用浏览器的第一语言作为默认的语言。当然你也可以自己配置语言 https://waline.js.org/guide/features/i18n.html 这个是非常基础的功能，在文档的很多地方都有描述。问问题之前可以多看看文档。
cc @37748846
